lib/Document/Renderer/DocumentRenderer.php

```diff
@@ -111,8 +111,14 @@ public function setContainerService(ContainerService $containerService)
      */
     public function render(Document\PageSnippet $document, array $attributes = [], array $query = [], array $options = []): string
     {
+        $eventAttributes = array_filter(
+            $attributes,
+            fn($key) => in_array($key, [EditmodeResolver::ATTRIBUTE_EDITMODE]),
```

Comment:
I think you forgot to add a `use` statement.

Reply:
thx!